programs/ssl/ssl_server2.c

```diff
@@ -1041,8 +1056,8 @@ int main( int argc, char *argv[] )
         }
         else if( strcmp( p, "renego_period" ) == 0 )
         {
-            opt.renego_period = atoi( q );
-            if( opt.renego_period < 2 || opt.renego_period > 255 )
+            if( sscanf( q, "%" SCNu64, &opt.renego_period ) != 1 ||
```

Comment:
I think the `SCNu64` macro is not part of C89 and since we are avoiding the use of `SIZE_T_MAX` because it is not standard, I don't think we should be using this either.